mtags/src/main/scala-2/scala/meta/internal/pc/completions/AmmoniteCompletions.scala

```diff
@@ -106,6 +109,76 @@ trait AmmoniteCompletions { this: MetalsGlobal =>
       }
   }
 
+  case class AmmoniteIvyCompletions(
+      select: Tree,
+      selector: List[ImportSelector],
+      pos: Position,
+      editRange: l.Range
+  ) extends CompletionPosition {
+
+    override def contribute: List[Member] =
+      try {
+
+        val query = selector.collectFirst {
+          case sel: ImportSelector if sel.name.toString().contains(CURSOR) =>
+            sel.name.decode.replace(CURSOR, "")
+        }
+
+        query match {
+          case Some(imp) =>
+            // TODO grab version from Ammonite file header if it exists
```

Comment:
Actually, this will be the same version `BuildInfo.scalaCompilerVersion` as we get info about it from the compiler

Reply:
Ah nice!

Reply:
From Ammonite - this is what I wanted to write :sweat_smile: